src/coreclr/interpreter/eeinterp.cpp

```diff
@@ -106,6 +106,11 @@ CorJitResult CILInterp::compileMethod(ICorJitInfo*         compHnd,
 
     try
     {
+        const char *className = NULL;
```

Comment:
We don't need this, we already have `DOTNET_InterpList` that enables dumping all the compiled methods.